Find a functionally equivalent code:android.net.MobileDataStateTracker.setEnableApn:COMMENT
Method Modifier: private     
Comment:/**
 * Internal method supporting the ENABLE_MMS feature.
 * @param apnType the type of APN to be enabled or disabled (e.g., mms)
 * @param enable {@code true} to enable the specified APN type,
 * {@code false} to disable it.
 * @return an integer value representing the outcome of the request.
 */

@@ -9,24 +9,23 @@
             loge("Ignoring feature request because could not acquire PhoneService");
             break;
         }
-        try {
-            if (enable) {
-                return mPhoneService.enableApnType(apnType);
-            } else {
-                return mPhoneService.disableApnType(apnType);
-            }
-        } catch (RemoteException e) {
-            if (retry == 0)
-                getPhoneService(true);
-        }
+    // try {
+    // if (enable) {
+    // return mPhoneService.enableApnType(apnType);
+    // } else {
+    // return mPhoneService.disableApnType(apnType);
+    // }
+    // } catch (RemoteException e) {
+    // if (retry == 0) getPhoneService(true);
+    // }
     }
     loge("Could not " + (enable ? "enable" : "disable") + " APN type \"" + apnType + "\"");
     return PhoneConstants.APN_REQUEST_FAILED;

